<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostWindow</class>
 <widget class="QMainWindow" name="HostWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HostWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="device_list">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>170</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="info_box">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>0</y>
      <width>345</width>
      <height>300</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="lcd_gpu">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Filled</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_gpu">
        <property name="text">
         <string>GPU clock</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="lcd_fun"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_fun">
        <property name="text">
         <string>Fan speed (RPM)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLCDNumber" name="lcd_voltage"/>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcd_memory"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_memory">
        <property name="text">
         <string>Memory clock</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_voltage">
        <property name="text">
         <string>Voltage</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLCDNumber" name="lcd_term"/>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBox_fun">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox_gpu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBox_memory">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_term">
        <property name="text">
         <string>Termal</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinBox_voltage">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinBox_term">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>51</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="label_gpu_max_min">
        <property name="text">
         <string>111 max
 11 min</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_mem_max_min">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_fan_max_min">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_voltage_max_min">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_te_max_min">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>gridLayoutWidget</zorder>
    <zorder>label_2</zorder>
    <zorder>verticalLayoutWidget</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
